.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "bierner.markdown-mermaid",
```

Comment:
I don't know how I feel about recommending Plugins to people who open this project with VS Code. I suppose it's fine, but I'm curious to know how if you have a reasoning behind it.

Reply:
It is a good practice as it speed up people with setting up the repo with any required tool. And as it is a recommendation, there is no push to use it or not ([Remix example](https://github.com/remix-run/remix/blob/main/.vscode/extensions.json)). 

As it is mostly related to `retype`, I first tried to find a relative extension but it wasn't any too. We can change it to a better extension if we have.